Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build the terraform-k8s binary
+FROM golang:1.13 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY version/ version/
+COPY workspacehelper/ workspacehelper/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o terraform-k8s main.go
+
+# Use distroless as minimal base image to package the terraform-k8s binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM alpine:3.12.1
+WORKDIR /
+COPY --from=builder /workspace/terraform-k8s .
+USER nonroot:nonroot
```

Comment:
I tried to run this image today. It built successfully, but in my cluster it gave me this error:

```
  Warning  Failed     0s (x4 over 53s)   kubelet, minikube  Error: failed to start container "terraform-sync-workspace": Error response from daemon: unable to find user nonroot: no matching entries in passwd file
```

I tried running it manually too. Looks like we'll have to choose another user, since the one specified in this Dockerfile does not exist.

```
$ docker images
REPOSITORY                             TAG     IMAGE ID      CREATED        SIZE
localhost/terraform-k8s-dev            latest  586fbea58e79  4 minutes ago  54.8 MB

$ docker run -ti --rm 586fbea58e79 /bin/bash
Error: unable to find user nonroot: no matching entries in passwd file
```

Reply:
Yeah this is probably due to the change from distroless to alpine. I will switch it to nobody that I know for a fact it exists in the alpine image.